sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -2131,6 +2131,11 @@ def test_sparse_metric_callable(csr_container):
     def sparse_metric(x, y):  # Metric accepting sparse matrix input (only)
         assert issparse(x) and issparse(y)
         return x.dot(y.T).toarray().item()
+        # result = x.dot(y.T)
+        # # TODO : remove once sparse matrix support is dropped (see #26418)
+        # if issparse(result):
+        #     return result.toarray().item()
+        # return result
```

Comment:
CI is green, so I guess this change isn't needed in the end.

```suggestion
```

Reply:
Ah, I think `_get_slice` returned the old sparse slicing behaviour, meaning this change is now no longer needed. Deleting!